OpenDreamShared/Dream/Procs/DreamProcOpcode.cs

```diff
@@ -248,4 +249,31 @@ public enum Type : byte {
             }
         }
     }
+
+    // Dummy class-as-namespace because C# just kinda be like this
+    public static class OpcodeVerifier
+    {
+        /// <summary>
+        /// Validates that the opcodes in DreamProcOpcode are all unique, such that none resolve to the same byte.
+        /// </summary>
+        /// <returns>True if there are duplicate opcodes, false if not</returns>
+        public static bool AreOpcodesInvalid() // FIXME: Can this be made into something done during compiletime? Like, *this code's* compiletime? >:/
+        {
+            // I'm not *too* satisfied with this boolean schtick, as opposed to throwing,
+            // but since we're in OpenDreamShared I want each executable to be able to do what they want with this information.
+            
+            ///Key is an int we're already using for an opcode
```

Comment:
Three slashes makes an XML comment which is not valid here
```suggestion
            // Key is an int we're already using for an opcode
```